samples/Extensibility/Program.cs

```diff
@@ -23,9 +23,9 @@
 // SIMPLE EXTENSIBILITY MODEL (INLINE STRATEGY)
 // ------------------------------------------------------------------------
 
-strategy = new CompositeStrategyBuilder()
+strategy = new ResiliencePipelineBuilder()
     // Just add the strategy instance directly
-    .AddStrategy(new MySimpleStrategy())
+    .AddStrategy(_ => new MySimpleStrategy())
```

Comment:
@martintmk This isn't compiling and I wasn't sure what the intended fix was supposed to be.

Reply:
We can do:

``` csharp
.AddStrategy(_ => new MySimpleStrategy(), new MySimpleStrategyOptions())
```

Reply:
Let's create empty `MySimpleStrategyOptions`